tests/PHPStan/Rules/Functions/data/array_filter_empty.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/** @var \stdClass[] $objects */
```

Comment:
please add a namespace to this file and move the testing code into a method of a class with a name of your choice.